api/deploy/ocp/scc.yaml

```diff
@@ -0,0 +1,43 @@
+---
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: "{{ .Prefix }}ceph-csi"
```

Comment:
make this name completely configurable? instead of just supporting the prefix?

Reply:
We could, but not sure it is helpful. This is a Ceph-CSI piece, so it should be fine to have `ceph-csi` as part of the name?

The API always offers an option to set a different name if users really want to. They can call `NewSecurityContextConstraints()` and set the `.Name` of the object.

Reply:
NewSecurityContextConstraints replace prefix with  deployer still we will get ceph-csi as  suffix I think it should be fine